--- src/ompl/src/ompl/CMakeLists.txt	2016-11-19 19:12:22.000000000 +0200
+++ src/ompl/src/ompl/CMakeLists.txt	2017-07-23 21:40:02.000000000 +0200
@@ -59,6 +59,8 @@
 else(MSVC OR IS_ICPC)
   add_library(ompl SHARED ${OMPL_SOURCE_CODE})
 endif(MSVC OR IS_ICPC)
+
+find_library(LZ4 lz4)
 target_link_libraries(ompl
   ${OMPL_LINK_LIBRARIES}
   ${Boost_DATE_TIME_LIBRARY}
@@ -67,7 +69,8 @@
   ${Boost_FILESYSTEM_LIBRARY}
   ${Boost_SYSTEM_LIBRARY}
   ${Boost_CHRONO_LIBRARY}
-  ${CMAKE_THREAD_LIBS_INIT})
+  ${CMAKE_THREAD_LIBS_INIT}
+  ${LZ4})

 if(MSVC)
   set_target_properties(ompl PROPERTIES VERSION "${OMPL_VERSION}" STATIC_LIBRARY_FLAGS "psapi.lib ws2_32.lib")
